src/view/com/profile/ProfileSubpageHeader.tsx

```diff
@@ -72,9 +72,11 @@ export function ProfileSubpageHeader({
     }
   }, [openLightbox, avatar])
 
+  const useMobileLayout = isMobile || isNativeTablet
```

Comment:
Small nit, let's use `isMobileLayout` for this just so it's more clear that it's actually a bool and not a hook. Mostly a skill issue probably, but when I saw `useMobileLayout` below I went 🤔 looking for a hook, heh.

Reply:
Damn yeah good call, was literally thinking that while typing but I couldn't come up with a better name

2 hardest problems in computer science etc etc